data/json/recipes/recipe_food.json

```diff
@@ -4932,24 +4933,16 @@
   },
   {
     "type": "recipe",
-    "activity_level": "LIGHT_EXERCISE",
+    "activity_level": "NO_EXERCISE",
     "result": "fried_egg_sandwich",
```

Comment:
this could probably do with using the suffix infrastructure to have one recipe where you cook the egg and make everything from scratch and one where you use already fried eggs. Forcing players to do the fried egg recipe first and then the sadnwich recipe adds a tiny bit of annoyance that we could do without

Reply:
Adding many of the same recipes is something I'd like to avoid.
Just like the code I didn't add above.

Now, recipes of the same product (different suffixes) are piled up, and I can't simplify them, and I can't fold them.
I hope that in the future, players will be allowed more freedom to use alternative ingredients, rather than adding massive products to the same recipe. (like PR), which requires C++ code to implement.

Reply:
Many sandwich recipes, both in the game and in real life, actually go through a two-step process: take something that can't be eaten raw and make it edible, then put it into slices of bread. Like "meat sandwiches".

So, I'm not actually subverting player behavior. Just made the fried egg sandwiches more like meat sandwiches.

Reply:
Many sandwich recipes, both in the game and in real life, actually go through a two-step process: take something that can't be eaten raw and make it edible, then put it into slices of bread. Like "meat sandwiches".

So, I'm not actually subverting player behavior. Just made the fried egg sandwiches more like meat sandwiches.

Reply:
Adding a recipe is really not much of a problem, they could even fit in a recipe category so that they can be collapsed in the menu